assets/javascripts/discourse/initializers/encrypt-drafts.js

```diff
@@ -107,5 +112,19 @@ export default {
         return _super.call(this, ...arguments);
       },
     });
+
+    UserDraftsStream.reopen({
```

Comment:
Can you use the plugin API for this?

Reply:
Of course, I will remove this usage and all other usages of `reopen` and `reopenClass` in the next commit.